services/worker-manager/src/providers/google.js

```diff
@@ -167,10 +168,6 @@ class GoogleProvider extends Provider {
   }
 
   async removeResources({workerPool}) {
-    // remove any remaining providerData when we are no longer responsible for this workerPool
-    await workerPool.modify(wt => {
-      delete wt.providerData[this.providerId];
-    });
```

Comment:
None of the four provider implementations actually use workerpools' providerData.  But google was the only one to try to remove that providerData in `removeResources`.  Because of how I structured `update_worker_pool_provider_data`, it's actually impossible to *remove* it, so I think if we were putting any data here then `removeResources` would just set it to `{}`.  But that's what we do in the setup, so this seemed redundant.

tl;dr: unused but maybe useful someday, so I made all providers look the same in the simplest way possible